WebHostLib/robots.py

```diff
@@ -0,0 +1,24 @@
+from WebHostLib import app
+from flask import abort
+import os
+import Utils
+from . import cache
+
+
+@cache.cached()
+@app.route('/robots.txt')
+def robots():
+    configpath = os.path.abspath("config.yaml")
```

Comment:
within the function, nothing of this is needed, just app.config["ASSET_RIGHTS"] suffices